src_docs/developer/01.ContributingOmegaT.md

```diff
@@ -1,28 +1,69 @@
+new features — you're more than welcome to join us!
 
-If you would like to submit a patch, the recommended procedure is as follows:
+The main development takes place in the `master` branch of our Git repository:  
+👉 [https://github.com/omegat-org/omegat](https://github.com/omegat-org/omegat)
 
-0. Consider opening a discussion on the developers' list first:
-  [http://lists.sourceforge.net/lists/listinfo/omegat-development](http://lists.sourceforge.net/lists/listinfo/omegat-development)
+Whether you're just getting started or already familiar with Git and Java, this guide will help you make your first
+contribution.
 
-1. Open a ticket for your change.
+---
 
-  * If it's a bug fix, create a Bug ticket:
-   [https://sourceforge.net/p/omegat/bugs/](https://sourceforge.net/p/omegat/bugs/)
+## Step-by-Step: How to Contribute
 
-  * If it's a new feature or enhancement, open a Request For Enhancement (RFE):
-   [https://sourceforge.net/p/omegat/feature-requests/](https://sourceforge.net/p/omegat/feature-requests/)
+### 1. Start a Conversation
 
-  * We don't use GitHub issues. Please don't use it!
+Before submitting code, we recommend starting a discussion on the **OmegaT Developers Mailing List**:  
+📬 [http://lists.sourceforge.net/lists/listinfo/omegat-development](http://lists.sourceforge.net/lists/listinfo/omegat-development)
 
-2. If you prefer to work with file-based patches, attach your patch to the
-   ticket.  If you prefer to work with pull requests, fork the OmegaT project on
-   GitHub and open a pull request.
+This gives everyone a chance to understand your idea, give feedback, or suggest alternatives.
 
-3. Patches/PRs will be reviewed by a core developer. If accepted, the changes
-   will be committed to master by the reviewer or an integration manager.
+---
 
+### 2. Open a Ticket
+
+Once your idea is clear, please open a ticket to track your proposed change:
+
+- 🐛 **For bug fixes**:  
+  Open a "Bug" ticket here:  
+  [https://sourceforge.net/p/omegat/bugs/](https://sourceforge.net/p/omegat/bugs/)
+
+- ✨ **For new features or improvements**:  
+  Open a "Request For Enhancement (RFE)" here:  
+  [https://sourceforge.net/p/omegat/feature-requests/](https://sourceforge.net/p/omegat/feature-requests/)
+
+> ⚠️ We do **not** use GitHub Issues. Please use SourceForge only!
+
+---
+
+### 3. Submit Your Code
+
+You can contribute your changes in one of two ways:
+
+- **Option A: Patch file**  
+  Create a patch (e.g., using `git diff`) and attach it to the ticket on SourceForge.
+
+- **Option B: Pull request (PR)**  
+  Fork the OmegaT project on GitHub and open a PR from your fork.
+
+---
+
+### 4. Code Review and Integration
+
+All submitted patches and pull requests will be reviewed by a core developer. If your change is accepted:
+
+- It will be merged into the `master` branch by the reviewer or an integration manager.
+- You’ll receive feedback, suggestions, or questions if any improvements are needed.
```

Comment:
One of the two options is that the code is not accepted, so I think we should change how we introduce them.

```suggestion
All submitted patches and pull requests will be reviewed by a core developer:

- If your change is accepted, it will be merged into the `master` branch by the reviewer or an integration manager.
- If any improvements are needed, you’ll receive feedback, suggestions, or questions.
```